astropy/coordinates/builtin_frames/itrs.py

```diff
@@ -77,7 +77,8 @@ def earth_location(self):
         """
         from astropy.coordinates.earth import EarthLocation
 
-        cart = self.represent_as(CartesianRepresentation)
+        cart = self.represent_as(CartesianRepresentation).without_differentials()
+        cart += self.location.get_itrs().cartesian
```

Comment:
Should this be `self.location.get_itrs(self.obstime)`?

Reply:
I guess it wouldn't hurt. I tested it using an `EarthLocation`, which has no time dependence. Can you think of a way to test with a location that would have a time dependence?

Reply:
On tests. It did pass all tests that use that attribute after I dropped the differentials. Do we need a test to specifically test this attribute?

Reply:
I guess I can add a simple test.